doc/tutorial/index.rst

```diff
@@ -0,0 +1,198 @@
+.. _tutorial:
+
+===============
+Sphinx tutorial
+===============
+
+In this tutorial you will build a simple documentation project using Sphinx,
+and view it in your browser as HTML.
+The project will include narrative, handwritten documentation,
+as well as autogenerated API documentation.
+
+The tutorial is aimed towards Sphinx newcomers
+willing to learn the fundamentals of how projects are created and structured.
+You will create a fictional software library to generate random food recipes
+that will serve as a guide throughout the process,
+with the objective of properly documenting it.
+
+To showcase Sphinx automatic documentation generation capabilities
```

Comment:
Note, *automatic* documentation generation with standalone Sphinx is as far as I know only available for Python.
How about having first a section on the manual documentation using domain directives (which indeed translates quite similarly to other domains), and then an additional section with autodoc for Python?
Thereby the underlying concepts are introduced and then the shorthands on top of that.

Reply:
It's an excellent point @jakobandersen, I was a bit misguided here. Will rephrase it so it makes more sense.